docs/source/AdministratorGuide/technologyPreviews.rst

```diff
@@ -8,17 +8,3 @@ The reason might be that it is not completely mature yet, or that it can be dist
 They are meant to stay optional for a couple of releases, and then they become the default.
 This page keeps a list of such technologies.
 
-M2Crypto
-========
-
-We aim at replacing the home made wrapper of openssl pyGSI with the standard M2Crypto library. It is by default disabled.
-You can enable it by setting the environment variable `DIRAC_USE_M2CRYPTO` to `Yes`.
-
-Possible issues
----------------
-
-M2Crypto (or any standard tool that respects TLS..) will be stricter than PyGSI. So you may need to adapt your environment a bit. Here are a few hints:
-
-* SAN in your certificates: if you are contacting a machine using its aliases, make sure that all the aliases are in the SubjectAlternativeName (SAN) field of the certificates
```

Comment:
This sounds like information to keep, somewhere in the administrator guide prepare server installation parts?

Reply:
How about the v7r1 and v7r2 release notes?

Reply:
Also here: https://dirac.readthedocs.io/en/integration/AdministratorGuide/ServerInstallations/InstallingDiracServer.html#server-certificates
It will be true for any server installation

Reply:
@chrisburr please add the info where Andre' is pointing. 

BTW, right now there are conflicts in this PR.